test/VerifySignedMessage.test.js

```diff
@@ -0,0 +1,433 @@
+        }
+      },
+      signature: '0x293eedbedbf8b41fd110c5e81e2df100bdc7cdf647a63c3f7973af2680b95e656b46b5eb0b67c86271245a913a9904f1e0553e1e47a16512782ed57cfc2673f71b',
+      signer: '0xf354C24DCE0f33E73C2150B5fC39B2d676a39a8f',
+      accountAddress: '0x044e9527429b01211b06B06885E6c5873f82156C',
+      functionName: 'metaDelegateCall',
+      signedParams: [
+        {
+          name: 'to',
+          type: 'address',
+          value: '0x53D468E719694f3e542Dda96a237Af08eb394f2C'
+        },
+        {
+          name: 'data',
+          type: 'bytes',
+          value: '0x3e3d6ada00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000400000000000000000000000000ff20817765cb7f73d4bde2e66e067e58d11095c2000000000000000000000000ba100000625a3754423978a60c9317c58a424e3d0000000000000000000000000000000000000000000000000de0b6b3a76400000000000000000000000000000000000000000000000000000007aad6342f69ccffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff',
+          callData: {
+            functionName: 'tokenToToken',
+            params: [
+              { name: 'bitmapIndex', type: 'uint256', value: '0' },
+              { name: 'bit', type: 'uint256', value: '1024' },
+              {
+                name: 'tokenIn',
+                type: 'address',
+                value: '0xfF20817765cB7f73d4bde2e66e067E58D11095C2'
+              },
+              {
+                name: 'tokenOut',
+                type: 'address',
+                value: '0xba100000625a3754423978a60c9317c58a424e3D'
+              },
+              {
+                name: 'tokenInAmount',
+                type: 'uint256',
+                value: '1000000000000000000'
+              },
+              {
+                name: 'tokenOutAmount',
+                type: 'uint256',
+                value: '2158161812220364'
+              },
+              {
+                name: 'expiryBlock',
+                type: 'uint256',
+                value: '115792089237316195423570985008687907853269984665640564039457584007913129639935'
+              },
+              { name: 'to', type: 'address' },
+              { name: 'data', type: 'bytes' }
+            ]
+          }
+        }
+      ]
+    }
+    brink.verifySignedMessage(message)
+  })
+
+  it('Verifies a correctly signed tokenToEth message', async function () {
+    const message = {
+      message: '0xd4d05ba90766cfae9de8348a026a188b7378fe818cf2f94bb51a9b1c93140529',
+      EIP712TypedData: {
```

Comment:
I don't think the EIP712TypedData object needs to be verified, these can probably safely be removed from tests...